sickbeard/providers/kat.py

```diff
@@ -72,6 +72,14 @@ def _doSearch(self, search_strings, search_mode='eponly', epcount=0, age=0, epOb
         results = []
         items = {'Season': [], 'Episode': [], 'RSS': []}
 
+#       select the correct category
+        if self.show.anime:
+            self.search_params.update({'category': 'anime'})
+            logger.log("search param 'category': 'anime' instead of 'tv' ", logger.DEBUG)
+        else:
+            self.search_params.update({'category': 'tv'})
+            logger.log("search param 'category': 'tv' ", logger.DEBUG)
+			
```

Comment:
Just do:
`self.search_params.update({'category': ('tv', 'anime')[self.show.anime]})`

But you also need to make the change to the rss update url when self.show.anime